.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,15 @@ Please label this PR with one of the following labels, depending on the scope of
 - Docs
 -->
 
+## What is the problem this PR solves?
```

Comment:
I always took this as the `Why is it important?` section. Should that section be removed in place of this section?

Reply:
I was looking at these 2 a bit different. This describes the problem itself without rating it on importance. But I agree, often this part will contain content around why it is important because it blocks xyz or fixes memory leak etc. 

I'll remove the "Why is it important?" section.